doc/user/content/manage/troubleshooting.md

```diff
@@ -10,104 +10,340 @@ aliases:
   - /ops/troubleshooting/
 ---
 
-This page describes several SQL queries you can run to diagnose performance
-issues with Materialize.
+This page describes several SQL queries you can run to diagnose performance issues with Materialize.
 
-{{< warning >}}
-The queries below reference [introspection relations](/sql/system-catalog/mz_internal/#replica-introspection-relations), which are subject to future changes without notice.
+## Mental model and basic terminology
 
-Due to the potential changes in introspection relations all relations below are created in the `mz_internal` schema, and it is not possible to create higher-level views dependent on them.
-{{< /warning >}}
+When you create a materialized view and issue a query, Materialize creates a so-called dataflow. A dataflow consists of instructions on how to respond to data input and to changes to that data. Once executed, the dataflow computes the result of the SQL query, polls the source for updates, and then incrementally updates the query results when new data arrives.
```

Comment:
This is imprecise. Dataflows can be created in three cases:

1. When creating a materialized view
2. When creating an index
3. When running a one-off (aka ad-hoc) query, but only if that query does not use the "fast path" execution strategy (i.e., looking at an existing index)

I would also avoid using the word "poll" as the system is not poll-based. I would prefer "waits for updates from the sources" or similar.